components/FilterBar/README.md

```diff
@@ -11,38 +11,10 @@ to switch between different view modes (grid, list), to sort and filter products
 ```
 import { FilterBar } from 'Templates/components';
```

Comment:
I see that you removed some content of this file. But it seems, the rest of the descriptions also don't match the actual component, since everything was moved to to the `Content` component. Not even the import here is correct :-P

Reply:
Corrected README